src/app/TideSearchApplication.cpp

```diff
@@ -429,7 +429,7 @@ vector<int> TideSearchApplication::getNegativeIsotopeErrors() const {
   return negative_isotope_errors;
 }
 
-vector<TideSearchApplication::InputFile> TideSearchApplication::getInputFiles(
+vector<InputFile> TideSearchApplication::getInputFiles(
```

Comment:
why TideSearchAppliation::InputFile to Input

Reply:
Because I move the InputFile struct out of TideSearchApplication so that it can be accessed by DIAmeter outside.